frontend/src/pages/NewAppointment.js

```diff
@@ -0,0 +1,142 @@
+import {Button, MenuItem, TextField} from '@mui/material';
+import {useEffect, useState} from 'react';
+import styled from 'styled-components/macro';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import {DatePicker, LocalizationProvider} from '@mui/lab';
+import moment from 'moment';
+import {getDoctors, submitAppointment} from '../service/DoctorApiService';
+
+export default function NewAppointment() {
+	const initialAppointment = {
+		date: null,
+		reasonForVisit: '',
+		notes: '',
+	};
+
+	const initialDoctor = {
+		id: '',
+		firstName: '',
+		lastName: '',
+		specialty: '',
+		street: '',
+		streetNumber: '',
+		postalCode: '',
+		city: '',
+		country: '',
+		phoneNumber: '',
+		mobileNumber: '',
+		emailAddress: '',
+		appointmentDto: initialAppointment,
+	};
+
+	const [newAppointment, setNewAppointment] = useState(initialAppointment);
+	const [doctor, setDoctor] = useState(initialDoctor);
+	const [doctors, setDoctors] = useState([]);
```

Comment:
just be careful, it can easily lead to errors, naming two variables so similarly. I usually try and name on of them i.e. allDoctors or singleDoctor, just to avoid mistyping and running into difficult to debug errors